common/perception_utils/include/perception_utils/perception_utils.hpp

```diff
@@ -102,6 +146,35 @@ inline double get2dIoU(const T1 source_object, const T2 target_object)
   return iou;
 }
 
+template <class T1, class T2>
+inline double get2dGeneralizedIoU(const T1 source_object, const T2 target_object)
```

Comment:
[MUST] What is difference with "get2dIoU" ?
I think this function is simplified by using "get2dIoU" or commonly used function.

Reply:
This function uses the area of Union, so if this function uses "get2dIoU", the area of Union is calculated twice, in "get2dIoU" and "get2dGeneralizedIoU".